packages/website/pages/pricing.js

```diff
@@ -132,96 +135,107 @@ const Card = props => {
   );
 };
 
-const PricingTiers = () => (
-  <>
-    <section id="section_plan_cards" className="sectional">
-      <div className="grid-middle">
-        <div>
-          <div className="column-content">
-            <Card
-              title="Free"
-              price="$0"
-              storageAllocation="5GiB storage"
-              storageOverageRate=""
-              callToAction="GET STARTED"
-              callToActionUrl="/account/payment?plan=free"
-            >
-              <li className="pricing-bullet-1">Easily store your data and make it available on IPFS</li>
-              <li className="pricing-bullet-2">
-                All data is replicated onto the Filecoin storage network for verifiability that your data is safe
-              </li>
-              <li className="pricing-bullet-3">
-                Use the platform&apos;s other services like w3name and w3link to build the next generation of apps
-              </li>
-            </Card>
+const PricingTiers = () => {
+  const [isEnterpriseRequestModelOpen, setIsEnterpriseRequestModelOpen] = useState(false);
+
+  return (
+    <>
+      <section id="section_plan_cards" className="sectional">
+        <div className="grid-middle">
+          <div>
+            <div className="column-content">
+              <Card
+                title="Free"
+                price="$0"
+                storageAllocation="5GiB storage"
+                storageOverageRate=""
+                callToAction="GET STARTED"
+                callToActionUrl="/account/payment?plan=free"
+              >
+                <li className="pricing-bullet-1">Easily store your data and make it available on IPFS</li>
+                <li className="pricing-bullet-2">
+                  All data is replicated onto the Filecoin storage network for verifiability that your data is safe
+                </li>
+                <li className="pricing-bullet-3">
+                  Use the platform&apos;s other services like w3name and w3link to build the next generation of apps
+                </li>
+              </Card>
 
-            <Card
-              title="Lite"
-              price="$3"
-              storageAllocation="15GiB storage"
-              storageOverageRate="+ $0.20/mo per additional GiB"
-              callToAction="CHOOSE THIS PLAN"
-              callToActionUrl="/account/payment?plan=lite"
-            >
-              <li className="pricing-bullet-1">
-                <i>Everything from the Free tier, plus...</i>
-              </li>
-              <li className="pricing-bullet-2">
-                Additional storage for personal usage or projects requiring lower data volumes
-              </li>
-            </Card>
+              <Card
```

Comment:
GitHub shows the space before the `<` a bit darker green, and only on this card. Not sure what that means, but there might be an extra space in there?

Reply:
Looks like VS Code fixed some formatting here.  As far as I can tell this is correct based on the other `<Card />`s